apps/blog/src/app/form.service.ts

```diff
@@ -7,7 +7,10 @@ import { Post } from './form/form.component';
   providedIn: 'root'
 })
 export class FormService {
-  repo$: AngularFireList<Post> = this.database.list('/posts');
+  all$: AngularFireList<Post> = this.database.list('/posts');
+  repo$: AngularFireList<Post> = this.database.list('/posts', (ref) => {
```

Comment:
Why is this form service thing is needed at all? Just use post.service for everything?